components/appbar/separators.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Separators
+page_title: AppBar - Separators
+description: Separate items in the Telerik AppBar for Blazor
+slug: appbar-separators
+tags: telerik,blazor,appbar,navbar,separator,spacer
+published: True
+position: 5
+---
+
+# Separators
+
+You can visually separate the items in the Telerik AppBar for Blazor. Depending on the needs of your application you can use of the following, or a combination of them.
```

Comment:
The introductory sentence should be self-sufficient to some extent, so that the reader finds out what the article is about, and if it's relevant to the reader's needs.

```suggestion
You can visually separate the items in the Telerik AppBar for Blazor. The component supports lines and empty space between its sections.
```